### PR TITLE
Feature merge glooctl

### PR DESCRIPTION
**Pull request by EItanya**
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/1172
resolves https://github.com/solo-io/gloo/issues/602
resolves https://github.com/solo-io/gloo/issues/722
resolves https://github.com/solo-io/gloo/issues/907
resolves https://github.com/solo-io/gloo/issues/932

### CODE REVIEW

File: changelog/v0.18.41/merge-cli.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+changelog:
+- type: FIX
+  description: add back the virtualservice create consul related flags
```
Comment (marcogschmidt) [inline changelog/v0.18.41/merge-cli.yaml]: Super nit: can we capitalize the first work in all the `description`s? It looks much better in the release change list .

Comment (ashleywang1) [reply changelog/v0.18.41/merge-cli.yaml]: Fixed in commit 791909e

**Pull request merged**